.github/workflows/smoke-test.yaml

```diff
@@ -1,29 +1,90 @@
-name: Smoke test automation-flavors
+name: Smoke test infra flavors
 run-name: >-
   ${{
-    format('Smoking testing automation-flavor version {0} on infra cluster {1}',
-      inputs.automation-flavors-version,
-      inputs.cluster
+    format('Smoke testing flavors on infra cluster {0} (prefix {1})',
+      inputs.cluster,
+      inputs.prefix
     )
   }}
 
 on:
   workflow_dispatch:
     inputs:
-      automation-flavors-version:
-        description: automation-flavors version, e.g. 0.10.2
-        required: true
-        default: "0.10.1"
       cluster:
         description: Infra cluster to run smoke test on (in acs-team-temp-dev project)
         required: true
-        default: "actions-pr-48"
+      prefix:
+        description: Prefix added to the clusters created for the smoke test (maximum 8 chars)
+        default: ist
+        required: true
 
 jobs:
   debug:
     runs-on: ubuntu-latest
     steps:
     - name: Show inputs
       run: |
-        echo "automation-flavors version: ${{ inputs.automation-flavors-version }}"
         echo "Cluster: ${{ inputs.cluster }}"
+        echo "Prefix: ${{ inputs.prefix }}"
+
+  smoke-test:
+    runs-on: ubuntu-latest
+    env:
+      KUBECONFIG: /home/runner/artifacts/kubeconfig
+      INFRA_TOKEN_PROD: ${{ secrets.INFRA_TOKEN }}
+      INFRA_TOKEN_DEV: ${{ secrets.INFRA_TOKEN_DEV }}
+      USE_GKE_GCLOUD_AUTH_PLUGIN: "True"
+    strategy:
+      matrix:
+        flavors: [
+          {name: "demo", "args": ""},
+          {name: "aks", "args": ""},
+          {name: "aro", "args": ""},
+          {name: "eks", "args": "user-arns=arn:aws:iam::393282794030:user/joey@stackrox.com"},
+          {name: "gke", "args": ""},
+          {name: "ibmroks", "args": ""},
+          {name: "ocp-4", "args": ""},
+          {name: "ocp-4-demo", "args": ""},
+          {name: "ocp-4-perf-scale", "args": ""},
+          {name: "ocp-3", "args": ""},
+          {name: "osd-on-aws", "args": ""},
+          {name: "osd-on-gcp", "args": ""},
+          {name: "qa-demo", "args": "main-image=quay.io/rhacs-eng/main:4.3.4"},
```

Comment:
> main:latest may be broken 

It will be broken a lot and inconsistent to reproduce. Do you know for which builds latest is tagged? (just master, tagged builds, dev branches, ...)